src/components/home/Home.tsx

```diff
@@ -27,14 +26,14 @@ function Home({ isRedirected, setIsRedirected }: Props) {
           <img src={teas} />
           <div className="info">
             <h2 className="home-h2">Home</h2>
-            <p>
+            <p className="info-desc">
               Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
-              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
-              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut
-              aliquip ex ea commodo consequat.
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim veniam, quis nostrud exercitation ullamco laboris
+              nisi ut aliquip ex ea commodo consequat.
             </p>
             <button className="infoButton">
```

Comment:
Could you remove this button from this Home section?

Reply:
I just removed that in my branch!